examples/single_agent.ipynb

```diff
@@ -0,0 +1,231 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example: Single Agent"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Install SUMO."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Setup SUMO=1.10.0\n",
+    "!apt-get update --fix-missing\n",
+    "!apt-get install -y libspatialindex-dev"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Install SMARTS."
```

Comment:
Not sure what is going on but every time I try to run this on Google Colab, the runtime crashes after this step and is unable to restart/reconnect. Only reuploading the notebook makes it able to connect again

Reply:
I had not tested these in particular, I think they were from the ipynb-1 branch. I will bring them over from there. I am not sure how compelling they will be as examples though.

Reply:
I have fixed this issue. It was checking out `develop`.